automated/linux/ota-rollback/aklite-callback.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-echo "${MESSAGE}" > /var/sota/ota.signal
+echo "${MESSAGE}" >> /var/sota/ota.signal
 echo "${RESULT}" > /var/sota/ota.result
```

Comment:
should this be '>>' to?

Reply:
RESULT is not used in the scripts, so it doesn't really matter.